logic/atrium-logic/src/commonMain/kotlin/ch/tutteli/atrium/logic/RangeAssertions.kt

```diff
@@ -0,0 +1,8 @@
+package ch.tutteli.atrium.logic
+
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.creating.AssertionContainer
+
+interface RangeAssertions {
+    fun <T : Comparable<T>> isInRange(container: AssertionContainer<T>, range: ClosedRange<T>): Assertion
```

Comment:
please keep toBeInRange, it's a legacy that some methods in other Assertions start with is. Using toBeInRange will simplify migration in the future. Thanks
```suggestion
    fun <T : Comparable<T>> toBeInRange(container: AssertionContainer<T>, range: ClosedRange<T>): Assertion
```